src/backend/plugins/video/warren_video/indicators/daily_video_views.py

```diff
@@ -1,25 +1,32 @@
-""""Warren video indicators."""
+""""Daily video views indicators."""
 
 from typing import List
 
 import pandas as pd
 from ralph.backends.http.lrs import BaseHTTP, LRSQuery
 from ralph.models.xapi.concepts.constants.video import RESULT_EXTENSION_TIME
+from ralph.models.xapi.concepts.verbs.scorm_profile import CompletedVerb
 from ralph.models.xapi.concepts.verbs.video import PlayedVerb
 from warren.base_indicator import BaseIndicator
 from warren.filters import DatetimeRange
 from warren_video.conf import settings as video_plugin_settings
-from warren_video.models import VideoViews
+from warren_video.models import DailyVideoViews
 
 
-class DailyVideoViews(BaseIndicator):
+class DailyVideoViewsIndicator(BaseIndicator):
```

Comment:
We already know that it's an indicator, it's not required to mention it as a suffix:

```suggestion
class DailyVideoViews(BaseIndicator):
```